README.md

```diff
@@ -1 +1,172 @@
-# logzio-azure-blob
+# Ship Azure Blob logs
+
+To simplify shipping of your Azure Blob logs,
+we provide an automated deployment process.
+
+At the end of this process,
+your Azure function will forward logs from Azure Blob Storage
```

Comment:
@imnotashrimp Because it's not just an Azure function maybe we can say `this integration will forward...`

Reply:
Changed and checked.